src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,45 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './CurrentUser.scss';
+import { getUsers } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number;
+  clearUser: () => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({ userId, clearUser }) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    getUsers(userId)
+      .then(userFromServer => setUser(userFromServer));
+  }, [userId]);
+
+  return (
+    <div className="CurrentUser">
+      {user ? (
+        <>
+          <h2 className="CurrentUser__title">
+            <span>
+              Selected user:
+              {' '}
+              {user?.id}
```

Comment:
you have already checked if user is not null

```suggestion
              {user.id}
```